core/src/data.rs

```diff
@@ -55,6 +55,27 @@ impl DataType {
             DataType::Timestamp(ts) => json!(ts.format("%+").to_string()),
         }
     }
+    
+    pub fn to_string(&self) -> String {
```

Comment:
I think this method should instead be an `impl Display for DataType`. That'd be useful in other places too!

Reply:
It deliberately isn't, actually -- we have such an `impl Display` already, but it quotes string literals. If we wish to use them as part of other strings (e.g., specifically, node names for group universes), we don't want quoting. Hence this additional method. (This was why a test failed earlier.)

Reply:
Ah... I'm... I'd actually argue that the `impl Display` shouldn't quote strings. Quoted strings is in the domain of `impl Debug`. It feels unfortunate to have two basically identical ways of printing a `DataType`...